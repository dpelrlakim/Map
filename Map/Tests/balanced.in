[]= a 0 0
[]= a -4 4
[]= a 4 -4
[]= a -6 6
[]= a -2 2
[]= a 2 -2
[]= a 6 -6
[]= a 1 -1
[]= a 5 -5
[]= a 7 -7
erase a 7
erase a 4
erase a 1
erase a 6
erase a 0
= b a
clear b
erase a -4
erase a -6
erase a 2
clear a
