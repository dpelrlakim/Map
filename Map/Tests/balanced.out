a: (0,0)
a: (-4,4) (0,0)
a: (-4,4) (0,0) (4,-4)
a: (-6,6) (-4,4) (0,0) (4,-4)
a: (-6,6) (-4,4) (-2,2) (0,0) (4,-4)
a: (-6,6) (-4,4) (-2,2) (0,0) (2,-2) (4,-4)
a: (-6,6) (-4,4) (-2,2) (0,0) (2,-2) (4,-4) (6,-6)
a: (-6,6) (-4,4) (-2,2) (0,0) (1,-1) (2,-2) (4,-4) (6,-6)
a: (-6,6) (-4,4) (-2,2) (0,0) (1,-1) (2,-2) (4,-4) (5,-5) (6,-6)
a: (-6,6) (-4,4) (-2,2) (0,0) (1,-1) (2,-2) (4,-4) (5,-5) (6,-6) (7,-7)
a: (-6,6) (-4,4) (-2,2) (0,0) (1,-1) (2,-2) (4,-4) (5,-5) (6,-6)
a: (-6,6) (-4,4) (-2,2) (0,0) (1,-1) (2,-2) (5,-5) (6,-6)
a: (-6,6) (-4,4) (-2,2) (0,0) (2,-2) (5,-5) (6,-6)
a: (-6,6) (-4,4) (-2,2) (0,0) (2,-2) (5,-5)
a: (-6,6) (-4,4) (-2,2) (2,-2) (5,-5)
a: (-6,6) (-4,4) (-2,2) (2,-2) (5,-5)
b: (-6,6) (-4,4) (-2,2) (2,-2) (5,-5)
a: (-6,6) (-4,4) (-2,2) (2,-2) (5,-5)
b:
a: (-6,6) (-2,2) (2,-2) (5,-5)
b:
a: (-2,2) (2,-2) (5,-5)
b:
a: (-2,2) (5,-5)
b:
a:
b:
